packages/snaps-execution-environments/webpack.config.js

```diff
@@ -17,8 +18,35 @@ module.exports = (_, argv) => {
     };
   }
 
+  const module = {
+    rules: [
+      {
+        test: /\.tsx?$/u,
```

Comment:
Is there a reason for us to support `tsx`?

Reply:
Not really, but this is the same config as before. It doesn't hurt to have it, but if you prefer I can remove it.

Reply:
Ah, didn't notice that. Let's just keep as-is.